### PR TITLE
[VEN-249 - VEN-250] Withdraw from vesting vaults

### PR DESCRIPTION
**Pull request by therealemjy**
## Jira ticket(s)

VEN-249, VEN-250

## Changes

- add query and mutation functions to handle withdrawing from vesting vaults
- add `WithdrawFromVestingVaultModal` component
- update query keys of queries related to fetching vaults to pass parameters as an object rather than spreading them
- set timezone to UTC when running tests
- set `REACT_APP_CHAIN_ID` to 97 (testnet) when running tests

### CODE REVIEW

File: src/pages/Vault/modals/WithdrawFromVestingVaultModal/Withdraw/index.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import { ConnectWallet, Spinner, LabeledInlineContent, PrimaryButton } from 'components';
+import { useStyles } from './styles';
+
+export interface WithdrawProps {
+  stakedTokenId: TokenId;
+  poolIndex: number;
+  handleClose: () => void;
+}
+
+const Withdraw: React.FC<WithdrawProps> = ({ stakedTokenId, poolIndex, handleClose }) => {
```
Comment (coreyar) [inline src/pages/Vault/modals/WithdrawFromVestingVaultModal/Withdraw/index.tsx]: nit: This component doesn't strictly follow our container patter

File: src/pages/Vault/modals/WithdrawFromVestingVaultModal/WithdrawalRequestList/index.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+import { useGetXvsVaultWithdrawalRequests } from 'clients/api';
+import { convertWeiToTokens } from 'utilities';
+import { ConnectWallet, Spinner, LabeledInlineContent } from 'components';
+import { useStyles } from './styles';
+
+export interface WithdrawalRequestListProps {
+  poolIndex: number;
+}
+
+const WithdrawalRequestList: React.FC<WithdrawalRequestListProps> = ({ poolIndex }) => {
```
Comment (coreyar) [inline src/pages/Vault/modals/WithdrawFromVestingVaultModal/WithdrawalRequestList/index.tsx]: This component doesn't strictly follow our container pattern

File: src/clients/api/queries/getXvsVaultWithdrawalRequests/index.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { GetXvsVaultWithdrawalRequestsInput, GetXvsVaultWithdrawalRequestsOutput } from './types';
+import formatToWithdrawalRequest from './formatToWithdrawalRequest';
+
+export * from './types';
+
+const getXvsVaultWithdrawalRequests = async ({
```
Comment (coreyar) [inline src/clients/api/queries/getXvsVaultWithdrawalRequests/index.ts]: This doesn't seem to represent withdrawal requests but rather when deposits are available for withdrawal. I think there is a better suited name than WithdrawlRequests

File: src/pages/Vault/modals/WithdrawFromVestingVaultModal/Withdraw/index.tsx (hunk 1 above)
Comment (therealemjy) [reply src/pages/Vault/modals/WithdrawFromVestingVaultModal/Withdraw/index.tsx]: Indeed!

File: src/pages/Vault/modals/WithdrawFromVestingVaultModal/WithdrawalRequestList/index.tsx (hunk 2 above)
Comment (therealemjy) [reply src/pages/Vault/modals/WithdrawFromVestingVaultModal/WithdrawalRequestList/index.tsx]: Indeed!

**Review by coreyar — APPROVED** [review]

**Pull request merged**